api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -160,7 +160,7 @@ default RdrWorkspace.StatusEnum toModelStatus(WorkspaceActiveStatus workspaceAct
   @ValueMapping(source = "TRUE", target = "YES")
   @ValueMapping(source = "FALSE", target = "NO")
   @ValueMapping(source = "PREFER_NO_ANSWER", target = "PREFER_NOT_TO_ANSWER")
-  RdrDisability toRdrDisability(Disability d);
+  RdrYesNoPreferNot toRdrDisability(Disability d);
```

Comment:
Not strictly necessary for this change, but the Disability entity field in the RDR yaml is exactly YesNoPreferNot, so we don't need it